it/test/helpers/ComponentSpecBase.scala

```diff
@@ -34,12 +37,17 @@ trait ComponentSpecBase extends TestSuite with CustomMatchers
   val mockPort = WiremockHelper.wiremockPort.toString
   val mockUrl = s"http://$mockHost:$mockPort"
 
+  val testSessionId = "xsession-12345"
+  implicit val hc: HeaderCarrier = HeaderCarrier(sessionId = Some(SessionId(testSessionId)))
+  implicit val ec: ExecutionContext = app.injector.instanceOf[ExecutionContext]
+
   def config: Map[String, String] = Map(
     "microservice.services.auth.host" -> mockHost,
     "microservice.services.auth.port" -> mockPort,
     "microservice.services.if.url" -> mockUrl,
     "microservice.services.des.url" -> mockUrl,
-    "useBusinessDetailsIFPlatform" -> "false"
+    "useBusinessDetailsIFPlatform" -> "false",
```

Comment:
this would not make any difference in ITs but we are now more or less migrated to IF platform as far as I know;

Reply:
Will remove it now